DevelopmentProposals/readme.md

```diff
@@ -0,0 +1,12 @@
+## Overview
+
+This directory contains all the development proposals for OpenSim.
+
+## How to create a proposal
+
+1. Create a new directory under directory ***DevelopmentProposals***.
+2. Copy ***template.md*** into the new directory, and rename it appropriately.
+3. Fill in the proposal details, and add any relevant code snippet files and media files to the directory.
```

Comment:
Let's add a soft guideline. (suggested < 1Mb). If size becomes an issue, we can move to a separate repo.